Form/Type/UpdateSelectFieldActionType.php

```diff
@@ -0,0 +1,82 @@
+
+class UpdateSelectFieldActionType extends AbstractType
+{
+    public const FIELD  = 'field';
+    public const ADD    = 'multiselect_add';
+    public const REMOVE = 'multiselect_remove';
+
+    public function __construct(private LeadFieldChoiceLoader $leadFieldChoiceLoader, private LeadFieldValuesChoiceLoader $leadFieldValuesChoiceLoader)
+    {
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $this->leadFieldChoiceLoader->setType($options['multiple']);
+        $this->leadFieldValuesChoiceLoader->setType($options['multiple']);
+        $fieldList = $this->leadFieldChoiceLoader->loadChoiceList()->getOriginalKeys();
+        if (null == $options['data'] && !empty($fieldList)) {
+            $flipFieldList = array_flip($fieldList);
+            $firstField    = reset($flipFieldList);
+            $this->leadFieldValuesChoiceLoader->setDefaultFieldId((int) $firstField);
+        } elseif (!empty($options['data']) && isset($options['data'][self::FIELD]) && $options['data'][self::FIELD] > 0) {
+            $this->leadFieldValuesChoiceLoader->setDefaultFieldId((int) $options['data'][self::FIELD]);
+        }
+        $builder->add(self::FIELD, ChoiceType::class, [
+            'label'         => 'mautic.plugin.multiselect_handling.field_action.managed_field',
+            'required'      => true,
+            'choice_loader' => $this->leadFieldChoiceLoader,
+            'constraints'   => [
+                new NotBlank(),
+            ],
+            'label_attr' => ['class' => 'control-label'],
+            'attr'       => [
+                'class'    => 'form-control',
+                'onchange' => 'Mautic.getOptionsFromField(this)',
+            ],
+            'multiple' => false,
+            'expanded' => false,
+        ])->add(self::ADD, ChoiceType::class, [
+            'label'         => $options['multiple'] ? 'mautic.plugin.multiselect_handling.field_action.multiselect_add' : 'mautic.plugin.multiselect_handling.field_action.select_add',
+            'required'      => false,
+            'choice_loader' => $this->leadFieldValuesChoiceLoader,
+            'label_attr'    => ['class' => 'control-label'],
+            'attr'          => [
+                'class' => 'form-control',
+            ],
+            'multiple' => $options['multiple'],
+            'expanded' => false,
+        ])->add(self::REMOVE, ChoiceType::class, [
+            'label'         => $options['multiple'] ? 'mautic.plugin.multiselect_handling.field_action.multiselect_remove' : 'mautic.plugin.multiselect_handling.field_action.select_remove',
+            'required'      => false,
+            'choice_loader' => $this->leadFieldValuesChoiceLoader,
+            'label_attr'    => ['class' => 'control-label'],
+            'attr'          => [
+                'class' => 'form-control',
+            ],
+            'multiple' => true,
+            'expanded' => false,
```

Comment:
We don't need to remove anything, as it's a select field. This could be removed.